fish-rust/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "fish-rust"
+version = "0.1.0"
+edition = "2021"
+
+
+[dependencies]
+widestring-suffix = { path = "./widestring-suffix/" }
+
+autocxx = "0.23.1"
+cxx = "1.0"
+errno = "0.2.8"
```

Comment:
If you only need to read the errno value you could use `std::io::Error::last_os_error().raw_os_error().unwrap()`. If you also need to write it, you will indeed need a crate like `errno`.